spec/models/article_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Article, type: :model do
+  subject { Article.new(title: "Title123", description: "Description12") }
+
+  it "is valid with valid attributes" do
+    expect(subject).to be_valid
+  end
+
+  it "is not valid without a title" do
+    subject.title=nil
```

Comment:
better practice is to use something like this 
`subject { Article.new(title: title, description: "Description12") }
`
```
context "when title is empty" do
  let(:title) { nil }
  it "is not valid" do
    expect(subject).to_not be_valid
  end
end
```